observability/prometheus/README.md

```diff
@@ -0,0 +1,27 @@
+# MicroShift Prometheus exporter
+
+A MicroShift Prometheus exporter to collect unique metrics from a MicroShift cluster
+
+## Prerequisites
+ - MicroShift installed and running
+ - Go version >= 1.21.11
+
+## Steps to run it
```

Comment:
Can we remove the numbering from the section? 

Let's just explain each step in a separate paragraph and there won't be a need to use tabulation too.

Reply:
sure, it's done